components/__tests__/fixtures/content/es/foo.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  foo: 'foo in español',
+  bar: '',
+  baz: null,
+  sections: [require('./bar')],
```

Comment:
Maybe, to make sure the filename matching is working properly, we should make this `require('./baz')` instead.